mobile/src/main/java/mycroft/ai/SettingsActivity.kt

```diff
@@ -247,20 +214,7 @@ class SettingsActivity : AppCompatPreferenceActivity() {
                     }
                 }
             } else if (preference is SwitchPreference) {
```

Comment:
Do we need this?

Reply:
There are currently no switch preferences, so this can be removed, along with the concept of a switch preference.

It felt like the kind of thing that would almost certainly be needed in the future, so I didn't remove it.

I'm working on another change (dependency upgrades/cleanup) but I could add in another commit to remove the unused preference type. It'll be easy enough to rewrite or restore if it's ever needed.